src/components/menu/header-messages.jsx

```diff
@@ -77,6 +107,7 @@ export class HeaderMessagesUnconnected extends React.Component {
         className="notebook-header-messages-container"
       >
         {content}
+        {showLoginModal}
```

Comment:
I think you should be able to just inject the login modal jsx properties directly in here, rather than using a variable intermediary. We did that for content because it could be one of many things, but I think we can just unconditionally dump in the `LoginModal` component. The visibility property will take care of whether to show it or not.